test/integration/Makefile

```diff
@@ -1,2 +1,3 @@
 e2e-local:
-	go test -v ./...
+	go test -v 2>&1 ./... | go-junit-report -set-exit-code > report.xml
```

Comment:
Few things here:
- Can we only generate the report when running the tests from workflow file? 
- It will be good to follow test file naming convention from other repos. The report will be downloaded from object storage and stored in ECP machine so, having the same generic name will overwrite the file when downloaded multiple times.
e.g. `"${timestamp}_terraform_test_report.xml"`

Reply:
since it run inside the playbook i don't think i can call directly inside workflow file 
i can add the file name generation here itself i.e 
e2e-local:
	timestamp=$(date +'%Y%m%d%H%M')
        report_filename="${timestamp}_linodego_test_report.xml"
	go test -v 2>&1 ./... | go-junit-report -set-exit-code > "$report_filename"